.dockerignore

```diff
@@ -1,2 +0,0 @@
-# ignore all during docker image build
-*
```

Comment:
`target/` and `.cargo/` dirs should remain docker-ignored IMHO as they do not need to be sent to docker daemon (e.g. in case of an image rebuild on a dev machine where such dirs are present).